### PR TITLE
removed ts-lint remark, added instructions for nyc

### PR DESCRIPTION
**Pull request by jalouxx**
Updating the description for `lb-nyc`. Solving #3227 
## Checklist

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: packages/build/README.md [hunk 1]
```diff
@@ -101,6 +100,42 @@ Now you run the scripts, such as:
...
+      ],
+      "reporter": [
+        "text",
+        "html"
+      ],
+      "exclude-after-remap": false
+    }
+    ```
+
+  - To run code coverage with every test execution update your package.json:
```
Comment (jannyHou) [inline packages/build/README.md]: > To run code coverage with every test execution update your package.json:

nitpick:

To run code coverage with every test execution, update your `package.json`:

File: packages/build/README.md [hunk 2]
```diff
@@ -101,6 +100,42 @@ Now you run the scripts, such as:
...
+  ```json
+    "coverage": "lb-nyc",
+    "test": "npm run coverage lb-mocha \"dist/__tests__\"",
```
Comment (raymondfeng) [inline packages/build/README.md]: `npm run coverage -- lb-mocha \"dist/__tests__\"`?

**Comment by dhmlau** [comment]
@jalouxx, would you like to address the review comments above? Thanks.

File: packages/build/README.md [hunk 3]
```diff
@@ -7,7 +7,6 @@ LoopBack 4 or other TypeScript modules, including:
   [`tsc`](https://www.typescriptlang.org/docs/handbook/compiler-options.html) to
   compile typescript files
 - lb-eslint: Run [`eslint`](https://typescript-eslint.io/)
-- lb-tslint: Run [`tslint`](https://github.com/palantir/tslint)
```
Comment (bajtos) [inline packages/build/README.md]: This is great.

In general, we try to keep commits small in scope and avoid mixing unrelated changes in one commit. Ideally, we prefer to have one pull request for each logical change.

Please:
- Extract this change into a standalone commit inside this pull request
- Optionally, move that commit into a new pull request, and keep this PR to discuss instructions for nyc. This way we can land the easy change quickly.

File: packages/build/README.md (hunk 2 above)
Comment (bajtos) [reply packages/build/README.md]: In loopback-next, we define `coverage` script to open the coverage report, and run the tests from a `precoverage` script. See here:

https://github.com/strongloop/loopback-next/blob/c8fb8058c375fe22b0c64988c7ff91c1024084ed/package.json#L43-L44

There is also a special script to run the tests & report the coverage to Coveralls:

https://github.com/strongloop/loopback-next/blob/c8fb8058c375fe22b0c64988c7ff91c1024084ed/package.json#L42

Here is how we run `lb-nyc` (please replace `node packages/build/bin/run-nyc` with `lb-nyc`):

https://github.com/strongloop/loopback-next/blob/c8fb8058c375fe22b0c64988c7ff91c1024084ed/package.json#L57-L58

I would like us to stay consistent and use the same approach in all places (`loopback-next` scripts in `package.json`, documentation for `@loopback/build`).

**Comment by bajtos** [comment]
@jalouxx What's the status of this pull request? Are you still keen to continue working on this improvement?

**Pull request closed without merging**